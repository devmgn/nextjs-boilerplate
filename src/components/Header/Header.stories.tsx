import type { Meta, StoryObj } from "@storybook/react";
import { Header } from ".";

const meta: Meta<typeof Header> = {
  component: Header,
};

export default meta;
type Story = StoryObj<typeof Header>;

export const Default: Story = {};
