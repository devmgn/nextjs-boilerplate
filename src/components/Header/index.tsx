export const Header: React.FC<React.ComponentPropsWithoutRef<'header'>> = (
  props,
) => {
  return <header {...props}>header</header>;
};
