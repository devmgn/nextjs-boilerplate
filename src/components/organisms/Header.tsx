const Header = () => {
  return <header>header</header>;
};

export default Header;
