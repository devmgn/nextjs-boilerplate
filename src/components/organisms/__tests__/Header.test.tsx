import { render } from '@testing-library/react';
import Header from '../Header';

describe('Header', () => {
  test('スナップショットテスト', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
