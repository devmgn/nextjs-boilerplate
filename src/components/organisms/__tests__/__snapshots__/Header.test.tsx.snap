// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header スナップショットテスト 1`] = `
<DocumentFragment>
  <header>
    header
  </header>
</DocumentFragment>
`;
